fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1,11 +1,23 @@
-OpenTracks is a sport tracking application that completely respects your privacy.
-
-Features:
-* Sport tracking: track your sport and outdoor activities
-* Photos and markers: mark interesting locations while tracking
-* Altitude gain and loss: via barometric sensor
-* Altitude shown in EGM2008 (above mean sea level); exported as WGS84
-* Voice announcements
-* Supports Bluetooth LE sensors: heart rate, speed and distance (cycling), cadence (cycling), and power meter (cycling)
-* Export data: export tracks either as KMZ (incl. photos), KML, or GPX
-* No Internet access: Internet is not used
+A sport tracking buddy that respects your privacy.
+
+sport and outdoor activities grants happiness, skill and confidence.
+
+Value your health by keeping track on your training.
+Records as you go running or walking, and gives you a bike computer with a bigger screen for cycling.
+Mark interesting locations along your way with pictures.
+Keep recorded statistics in great detail for analysis.
+Share only the data you want others to have.
+
+* Voice announcements.
+* Supports Bluetooth LE sensors: heart rate, speed and distance (cycling), cadence (cycling), and power meter (cycling).
+* Altitude gain and loss: via barometric sensor.
+* Altitude shown in EGM2008 (above mean sea level); exported as WGS84.
+* Export data as tracks either as KMZ (incl. photos), KML, or GPX.
+* No Internet access or extra permissions.
+* Dark and light theme, respecting system settings.
+* No ads.
+
+Libre software / Free Software / Open Source
+meaning you can use, study, change and share the source code.
+https://github.com/OpenTracksApp/OpenTracks
```

Comment:
Why put the link here?
It is shown in F-Droid anyhow below.

Reply:
I was thinking it was better for the Play listing, but it is available at the bottom there too.
It doesn't work for searches either, so removing :)